### PR TITLE
[no-issue]: Move period type utils to tsutils

### PR DESCRIPTION
**Pull request by kael89**
Moving a few more of those across

### CODE REVIEW

File: packages/tsutils/src/period/period.ts [hunk 1]
```diff
@@ -19,3 +34,111 @@ export const PERIOD_TYPES = {
   QUARTER, // e.g. '2018Q1'
   YEAR, // e.g. '2018'
 } as const;
+
+const NON_NUMERIC_PERIOD_TYPES = [WEEK, QUARTER];
```
Comment (kael89) [inline packages/tsutils/src/period/period.ts:38]: Everything here (except for types) is copied over from `period.js` in `@tupaia/utils`

**Comment by kael89** [comment]
@rohan-bes now that you are in the review flow, you may also want to have a look at this one too 🙃 

All good if you are already aware of it - it's not urgent! Just a bump in case it got lost in a pile of emails and notifications 😄

**Review by rohan-bes — APPROVED** [review]
Awesome stuff, thanks @kael89 💜 

Just a heads up, that we're planning on converting `@tupaia/utils` into a TS/JS hybrid package soon. At that point we're gonna combine `@tupaia/utils` and `@tupaia/tsutils`. It should make these kinds of changes heaps easier, since you don't need to update imports, just convert the file to TS 😁

**Comment by kael89** [comment]
> 

Ah amazing, that makes a lot of sense! I'll hold off on further conversions for now then.

Happy to help convert the rest of utils to TS after that happens 🙂

**Pull request merged**